game-frontend/package-lock.json

```diff
@@ -11686,6 +11702,15 @@
         "tslib": "^1.0.0"
       }
     },
+    "react-update": {
```

Comment:
@sir-marc When reverting always also revert the lock file. This lock file contains entries which shouldn't be there. (such as this one)